### PR TITLE
Create Docker file

### PR DESCRIPTION
**Pull request by Lev-Stambler**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/step-2020-a-team-has-no-name/api-image', '.' ]
+images:
+- 'gcr.io/step-2020-a-team-has-no-name/api-image'
```
Comment (RolandNaijuka) [inline cloudbuild.yaml]: the project name is ```step-alr-2020```. It was changed

Comment (Lev-Stambler) [reply cloudbuild.yaml]: Oh Google Cloud?

Comment (Lev-Stambler) [reply cloudbuild.yaml]: Oh I see

Comment (Lev-Stambler) [reply cloudbuild.yaml]: Good catch. Changed the repository to `step-alr-2020`

**Review by RolandNaijuka — APPROVED** [review]
The docker file is on point. Thanks for setting it up.

**Review by cwlarson-google — APPROVED** [review]

**Pull request merged**